include/openthread/commissioner.h

```diff
@@ -110,6 +110,20 @@ typedef struct otCommissioningDataset
     bool mIsJoinerUdpPortSet : 1; ///< TRUE if Joiner UDP Port is set, FALSE otherwise.
 } otCommissioningDataset;
 
+#define OT_PSKD_MAX_SIZE 32 ///< Size of a Joiner PSKd (bytes)
+/**
+ * This structure represents a Joiner Info.
+ *
+ */
+typedef struct otJoinerInfo
+{
+    otExtAddress mEui64;                     /// Joiner eui64
+    char         mPsk[OT_PSKD_MAX_SIZE + 1]; /// Joiner pskd
+    uint32_t     mExpirationTime;            /// Joiner expiration time
```

Comment:
nit: use `///<` instead of `///` for doxygen?
also how about updating the comment for `mExpirationTime` to mention the time unit :  `///< Joiner expiration time in msec``